extensions/opentelemetry/runtime/src/main/java/io/quarkus/opentelemetry/runtime/exporter/otlp/LateBoundBatchSpanProcessor.java

```diff
@@ -18,7 +18,7 @@
 public class LateBoundBatchSpanProcessor implements SpanProcessor {
```

Comment:
Is this change in this file intended to be part of the change? 

If so, it looks sort of unrelated to the reported and should probably be a separate commit